EvidenceApi/V1/Gateways/DocumentsApiGateway.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using EvidenceApi.V1.Gateways.Interfaces;
+using EvidenceApi.V1.Infrastructure;
+using EvidenceApi.V1.Boundary.Request;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using EvidenceApi.V1.Domain;
+using System.Net.Http.Headers;
+
+namespace EvidenceApi.V1.Gateways
+{
+    public class DocumentsApiGateway : IDocumentsApiGateway
+    {
+        private readonly HttpClient _client;
+        public DocumentsApiGateway(HttpClient httpClient)
+        {
+            _client = httpClient;
+        }
+        public async Task<Claim> GetClaim(ClaimRequest request)
```

Comment:
Looking at this overall, I think this method should be `CreateClaim`. By convention, `Get` suggests you're fetching an existing resource, when in fact you're creating one.